docker-compose.test.yml

```diff
@@ -40,7 +40,7 @@ services:
       service: service_base
     volumes_from:
       - static
-    command: py.test inspirehep tests/unit
+    command: ./run_tests.sh
```

Comment:
Why not just `py.test inspirehep tests/unit && make -C docs html`? The ASCII art in `./run-tests.sh` adds no value to the report.